index.js

```diff
@@ -57,3 +63,17 @@ function tryParseFloat(value) {
 	}
 	return result;
 }
+
+function getTypeFromUnit ( unit ) {
+	if ( cssAngleUnits.indexOf(unit) !== -1 ) {
+		return 'angle';
+	} else if ( cssResolutionUnits.indexOf(unit) !== -1 ) {
```

Comment:
As you are returning early in each "if" statement, you actually don't need "else" anywhere in this function. Just a series of "if" statements should suffice. For example:
```js
if (condition1) {
    return 'foo';
}
if (condition2) {
    return 'bar';
}
return 'baz';
```